jscripts/post.js

```diff
@@ -116,7 +117,44 @@ var Post = {
 	{
 		document.input.attachmentaid.value = aid;
 		document.input.attachmentact.value = action;
+	},
+
+	initAttachments: function()
+	{
+		var fileUpload = $("input[type='file']");
+		fileUpload.attr("multiple","multiple");
+		$('form').on('submit', Post.checkAttachments);
+	},
+
+	checkAttachments: function()
+	{
+		var files = $("input[type='file']");
+		var file = files.get(0);
+		if (!file) 
+			return true;
```

Comment:
Please put this line in curly braces (`{` `}`) just to maintain coding style consistency.